aten/src/ATen/InferSize.h

```diff
@@ -25,6 +25,8 @@ inline void infer_size_impl(
   // N.B. this is an index, not a sym dim!
   std::optional<int64_t> infer_dim;
   for (int64_t dim = 0, ndim = shape.size(); dim != ndim; dim++) {
+    // We can avoid failing on unbacked shape[dim] and assert that it is >=0
+    // following python behaviour.
```

Comment:
I don't see how this comment relates to the code underneath

Reply:
when shape[dim] is unbacked we will fail when we do the check a later PR 
address this by adding Guard_or_false. 
how do you suggest that i rephrase the comment. ?

Reply:
I dont there is need for todo, and issue number since next PR is already up and have the change I can remove the comment completly